### PR TITLE
#511 Links to candidate page detail

### PR DESCRIPTION
**Pull request by lloback**
This PR deals with links to Candidade Detail page.

The links were added from:

- from Applications (applicationsFull)

Stages:
-  Review
-  Shortlisted
-  Selected
-  Recommended
-  Handover

### CODE REVIEW

File: src/views/Exercises/Show/ApplicationFull.vue [hunk 1]
```diff
@@ -94,7 +94,11 @@
                 Full Name
               </dt>
               <dd class="govuk-summary-list__value">
-                {{ application.personalDetails.fullName }}
+                <RouterLink
+                  :to="{ name: 'candidates-view', params: { id: application.id } }"
```
Comment (warrensearle) [inline src/views/Exercises/Show/ApplicationFull.vue:98]: The ID to use here is `application.userId`. Of course!

File: src/views/Exercises/Stages/HandoverList.vue [hunk 2]
```diff
@@ -40,7 +40,13 @@
       >
         <template #row="{row}">
           <TableCell>{{ row.application.referenceNumber }}</TableCell>
-          <TableCell>{{ row.candidate.fullName }}</TableCell>
+          <TableCell>
+            <RouterLink
+            :to="{ name: 'candidates-view', params: { id: row.id } }"
```
Comment (warrensearle) [inline src/views/Exercises/Stages/HandoverList.vue]: The id to use here is `row.candidate.id`. Of course!

File: src/views/Exercises/Show/ApplicationFull.vue (hunk 1 above)
Comment (lloback) [reply src/views/Exercises/Show/ApplicationFull.vue:98]: done.

File: src/views/Exercises/Stages/HandoverList.vue (hunk 2 above)
Comment (lloback) [reply src/views/Exercises/Stages/HandoverList.vue]: Fixed this everywhere on STAGES. Thanks.

File: src/views/Exercises/Show/Applications.vue [hunk 3]
```diff
@@ -56,18 +56,19 @@
             </router-link>
           </th>
           <td class="govuk-table__cell">
-            <span v-if="application.personalDetails">{{ application.personalDetails.fullName }}</span>
+            <span v-if="application.personalDetails">
+              <RouterLink
+                :to="{ name: 'candidates-view', params: { id: application.id } }"
```
Comment (warrensearle) [inline src/views/Exercises/Show/Applications.vue:61]: This should use `application.userId` instead of `application.id`

Comment (lloback) [reply src/views/Exercises/Show/Applications.vue:61]: done.

**Pull request merged**